### PR TITLE
Fixes to get app functioning on AWS Amplify

### LINKED ISSUE #225: Get TW App working on AWS Amplify

### PR DESCRIPTION
**Pull request by rpeterman-gp**
Closes #225 
- Gets app working in AWS Amplify deployments.

## To Review

- [x] Go to deploy env for this branch: https://feat-aws-amplify-deploys-01.dtaq9a3yfwk0i.amplifyapp.com/
- [x] Navigate around site and ensure all page types load.
- [x] Do a search.
- [x] Sign up for a newsletter.
- [x] Dance in your chair! 🕺

### CODE REVIEW

**Review by brandonhundt — APPROVED** [review]
LGTM. Nice work @rpeterman-gp :shipit:

File: store/reducers/aliasData.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const aliasData = (state: State = {}, action: AnyAction) => {
 
   switch (action.type) {
     case HYDRATE:
-      return { ...action.payload.aliasData, ...state };
+      return { ...state, ...action.payload.aliasData };
```
Comment (brandonhundt) [inline store/reducers/aliasData.ts:21]: Is this the final issue you had to resolve? flipping the loading state?

Comment (rpeterman-gp) [reply store/reducers/aliasData.ts:21]: Part of it. This was actually reverting something I was trying. Main issue was the way the api endpoints where returning (actually not returning) responses.

File: pages/api/category/[id]/index.ts [hunk 2]
```diff
@@ -52,12 +52,11 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
         stories
       };
 
-      res.status(200).json(apiResp);
-      res.end();
-    } else {
-      res.status(404);
+      return res.status(200).json(apiResp);
```
Comment (rpeterman-gp) [inline pages/api/category/[id]/index.ts:55]: Changes like this were the key fixes to the last of the issues.

Comment (brandonhundt) [reply pages/api/category/[id]/index.ts:55]: Ah, thats a good catch. Is this something worth writing up so others don't have to feel so much pain in the future?

**Pull request merged**